troop/models/participant.py

```diff
@@ -120,6 +120,10 @@ def age_section_order(cls):
             ]
         )
 
+    def get_age_section_display(self):
```

Comment:
Sorry, why do we need this?

Reply:
Better fix: https://github.com/packmas13/registration/pull/19/commits/543c90c9cea9f1f5ab3da5ce943c0135ead00fda